interview-form.html

```diff
@@ -0,0 +1,173 @@
+<html>
+  <head>
+    
+    <!-- HTML Meta Tags -->
+    <title>PI Club | Interview Form</title>
+    <meta name="description" content="Seats for PI Club are Available, Apply Now.">
+
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="https://forms.pi-club.ml/sample">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="PI Club | Interview Form">
+    <meta property="og:description" content="Seats for PI Club are Available, Apply Now.">
+    <meta property="og:image" content="https://pi-club.ml/logos/main.png">
+
+    <!-- Twitter Meta Tags -->
+    <meta name="twitter:card" content="summary_large_image">
+    <meta property="twitter:domain" content="forms.pi-club.ml">
+    <meta property="twitter:url" content="https://forms.pi-club.ml/sample">
+    <meta name="twitter:title" content="PI Club | Interview Form">
+    <meta name="twitter:description" content="Seats for PI Club are Available, Apply Now.">
+    <meta name="twitter:image" content="https://pi-club.ml/logos/main.png">
+
+    <!-- Meta Tags Generated via https://opengraph.dev -->
+
+    <link rel="stylesheet" href="./css/form.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Fira+Code&display=swap" rel="stylesheet">
+
+
+    <style>
+      body {
+        padding: 15vh 15vw;
+      }
+
+      @media (max-width: 400px) {
+        body {
+          padding: 5vh 5vw;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <form action="#" id="sample">
+      <h1 class="form-name">Form Name</h1>
+
+      <div id="name" class="form-input-group">
+        <label for="name-input">Name</label>
+        <!--why the weird span? cuz CSS can't be fucking serious http://forums.mozillazine.org/viewtopic.php?f=25&t=291007&start=0&st=0&sk=t&sd=a-->
+        <span class="text-input">
+          <input placeholder="Enter your full name" type="text" name="name" id="name-input">
+        </span>
+        <div class="error-text">Your name is required</div>
+      </div>
+
+      <div id="email" class="form-input-group">
+        <label for="email-input">Email</label>
+        <span class="text-input">
+          <input placeholder="Enter your email" type="text" name="email" id="email-input">
+        </span>
+        <div class="error-text">Email is not valid</div>
+      </div>
+
+      <div id="github" class="form-input-group">

```

Comment:
This can be a lot more consistent. For example, the label says "Github" (which is good). The placeholder says "Github User URL". The required message says "Github User Profile Link". The invalid message says "Github User Profile".
I suggest the label remains the simple "Github" but we pick on something for the placeholder, required message, and invalid message.
I vote for "github profile link"